modules/core/server/routes/core.server.routes.js

```diff
@@ -4,13 +4,20 @@
  * Module dependencies.
  */
 var path = require('path'),
+    redirect = require("express-redirect"),
```

Comment:
https://www.npmjs.com/package/express-redirect

> _This package isn't actively maintained anymore. It was written for express v3.x, but still seems to work with express v4.x. Use it at your own risk._

> _last publish 3 years ago_

Neat library but I don't think it's a good idea to add it. :-/

Reply:
Plugged in another one.

Reply:
I'm not really sure if saving 3-6 lines of code is really worth external dependency?

https://github.com/Trustroots/trustroots/pull/877#discussion_r236080391

Reply:
Well that first one showed up when I googled redirect and express.
If it's 3-6 lines we should not use an external dep.